api/qanpb/object_details.proto

```diff
@@ -60,6 +60,20 @@ service ObjectDetails {
       body: "*"
     };
   }
+  // SchemaByQueryID returns schema for given queryID and serviceID.
+  rpc SchemaByQueryID(SchemaByQueryIDRequest) returns (google.protobuf.StringValue) {
```

Comment:
Not sure that using `google.protobuf.StringValue` as a response is a good idea. There is no way to extend it in the future.

Reply:
There is already QueryExists which also directly returning bool (also done by me). Name of this one is SchemaByQueryID so I am not sure why it should be extended in future, because for anything else different method should be used. 

Anyway I will change it for sure when its requested.